### ISSUE TITLE
VIC UI installer script for Windows won't work under a certain condition

### ISSUE THREAD

**Issue by jooskim** [vmware/vic#4279]
**User Statement:**
As a UI engineer, I want to ensure the UI plugin installer works across all supported environments.

**Details:**
The plugin installer should work across all supported environments whether it's a vCSA or vCenter 6.5 instance, but the H5 Client plugin won't install against vCenter 6.5 without a web server serving the plugin zip file. Installing without a web server involves copying the plugin contents directly into the plugin cache folder of the vSphere (Web) Client, and this has worked for:

1. Flex Client 6.0 (both VCSA and vCenter)
2. Flex Client 6.5 (both VCSA and vCenter)
3. H5 Client (VCSA)

, but not H5 Client on vCenter 6.5. This approach is not recommended by the HTML SDK team and they suggested that we follow the standard plugin deployment procedure where there is a perpetual web server hosting the plugin zip file. To date, we don't have a web server for this use case, but soon once #4254 is done the zip file can be served inside the OVA appliance that also hosts Harbor and Admiral.

**Acceptance Criteria:**
1. Host the plugin zip file inside the OVA appliance.
2. Simplify the UI installation procedure

**Comment by mhagen-vmware** [comment]
@jooskim Is this issue still valid or closed with #4277 ?

**Comment by jooskim** [comment]
@mhagen-vmware This is a separate issue and will be addressed by the OVA work that Fabio is working on.

**Comment by jooskim** [comment]
Closed via #4254
